TFT/src/User/API/Temperature.c

```diff
@@ -122,7 +126,7 @@ void heatSetUpdateSeconds(uint8_t seconds)
   if (infoMachineSettings.autoReportTemp && !heat_update_waiting)
   {
     heat_update_waiting = true;
-    storeCmd("M155 ");
+    storeCmd("M155\n");
   }
```

Comment:
Hello, There should not be '\n', because in the `sendQueueCmd()` function, the latest temperature update time will be obtained and sent in the form of "M155 Sxx\n"
https://github.com/bigtreetech/BIGTREETECH-TouchScreenFirmware/blob/ce2ea18b94abe8e94b80e69e405999ded1b8c73a/TFT/src/User/API/interfaceCmd.c#L524

Adding '\n' to this gcode means that `"M155\n"` will be sent directly to Marlin, and the default `'S'` value of Marlin is 0. So automatic temperature reporting will never be available in this way,

Reply:
my bad, I will rollback the change

Reply:
I'm doing it